component.css

```diff
@@ -108,19 +108,19 @@
       left:0;
       bottom: 0;
       z-index:20;
-      width:17rem;
+      width:auto;
       display: flex;
       align-items: center;
       
   }
   .sidebar-list-item
   {
-      font-size: 2rem;
-      list-style: none;
-      padding: 0.5rem;
-      font-weight: 500;
-      align-self: center;
-      /* margin-top:2rem; */
+    font-size: 1.8rem;
+    list-style: none;
+    padding: 0.3rem;
```

Comment:
you can use px for less than 1rem

Reply:
to be consistent with length we can use rem,

Reply:
I have used px only in box-shadow property